src/azul/service/service_config/request_config.json

```diff
@@ -53,46 +53,46 @@
   },
   "manifest": {
     "bundles": {
-      "bundle_uuid": "uuid",
-      "bundle_version": "version"
+      "bundle.uuid": "uuid",
+      "bundle.version": "version"
     },
     "contents.files": {
-      "file_content_type": "content-type",
-      "file_name": "name",
-      "file_sha256": "sha256",
-      "file_size": "size",
-      "file_uuid": "uuid",
-      "file_version": "version",
-      "file_indexed": "indexed",
-      "file_format": "file_format"
+      "file.content_type": "content-type",
+      "file.name": "name",
+      "file.sha256": "sha256",
+      "file.size": "size",
+      "file.uuid": "uuid",
+      "file.version": "version",
+      "file.indexed": "indexed",
+      "file.format": "file_format"
     },
     "contents.cell_suspensions": {
-      "total_estimated_cells": "total_estimated_cells"
+      "cell_suspension.estimated_cell_count": "total_estimated_cells"
     },
     "contents.protocols": {
-      "instrument_manufacturer_model": "instrument_manufacturer_model",
-      "library_construction_approach": "library_construction_approach"
+      "protocol.instrument_manufacturer_model": "instrument_manufacturer_model",
+      "protocol.library_construction_method": "library_construction_approach"
     },
     "contents.projects": {
-      "document_id": "document_id",
-      "institutions": "institutions",
-      "laboratory": "laboratory",
-      "project_shortname": "project_shortname",
-      "project_title": "project_title"
+      "project.document_id": "document_id",
+      "project.institutions": "institutions",
+      "project.laboratory": "laboratory",
+      "project.project_short_name": "project_shortname",
+      "project.project_title": "project_title"
     },
     "contents.specimens": {
-      "biological_sex": "biological_sex",
-      "specimen_id": "biomaterial_id",
-      "specimen_document_id": "document_id",
-      "disease": "disease",
-      "donor_biomaterial_id": "donor_biomaterial_id",
-      "donor_document_id": "donor_document_id",
-      "genus_species": "genus_species",
-      "organ": "organ",
-      "organ_part": "organ_part",
-      "organism_age": "organism_age",
-      "organism_age_unit": "organism_age_unit",
-      "preservation_method": "preservation_method"
+      "specimen.sex": "biological_sex",
+      "specimen.biomaterial_id": "biomaterial_id",
+      "specimen.document_id": "document_id",
+      "specimen.diseases": "disease",
+      "specimen.donor_biomaterial_id": "donor_biomaterial_id",
+      "specimen.donor_document_id": "donor_document_id",
+      "specimen.genus_species": "genus_species",
+      "specimen.organ": "organ",
```

Comment:
Organ and organ parts are special because they could come from cell line, organoid or specimen. This conflation will go away soon with @danielsotirhos's samples PR. For now, we should leave `organ` and `organ_parts` under `specimen` even if that isn't always accurate.